YesterdayApi/Startup.cs

```diff
@@ -23,6 +25,8 @@ public void ConfigureServices(IServiceCollection services)
       services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_1);
       services.AddMapper();
       services.AddSwagger();
+      services.AddDbContext<YesterdayContext>(opt =>
```

Comment:
For better readibility create static function Add(extensionName), the rest looks fine for me